.github/workflows/build_linux_packages.yml

```diff
@@ -198,3 +173,21 @@ jobs:
         with:
           path: ${{ env.CACHE_DIR }}
           key: linux-build-packages-manylinux-v4-${{ inputs.amdgpu_families }}-${{ github.sha }}
+
+      - name: Upload job metrics to Redshift database
+        if: always()
```

Comment:
I may have missed this in my prior review, but we should set this condition to only run if the repository is ROCm/TheRock, where the secrets are set. Either that or the script should fail gracefully when the secrets are missing.